service_systemd_linux.go

```diff
@@ -67,6 +69,10 @@ func (s *systemd) Platform() string {
 	return s.platform
 }
 
+func (s *systemd) ConfigPath() (string, error) {
+	return s.configPath()
```

Comment:
As I previously mentioned, don't just wrap a private method, rename the existing one.

Reply:
Done